README.md

```diff
@@ -119,6 +119,7 @@ The least we can do is to thank them and list some of their accomplishments here
 #### 2022
 
 * [Filip Hrisafov](https://github.com/filiphr) contributed the [JSON Argument Source](https://junit-pioneer.org/docs/json-argument-source/) support (#101 / #492)
+* [Rob Spoor](https://github.com/robtimus) added registration of Jackson modules for use in `@JsonSource`, `@JsonFileSource` and `@JsonClasspathSource`. This allows modules like `JavaTimeModule` to be used (#624 / #629)
```

Comment:
```suggestion
* [Rob Spoor](https://github.com/robtimus) added registration of Jackson modules (e.g. `JavaTimeModule`) JSON sources (#624 / #629)
```

Reply:
I felt that it didn't read really well. I've set changed it to "added registration of Jackson modules (...) *for* JSON sources"

Reply:
Yeah, my proposal didn't grammar. 😉